auth-tokens/src/main/java/com/palantir/tokens/auth/UnverifiedJsonWebToken.java

```diff
@@ -44,9 +45,10 @@
 @ImmutablesStyle
 public abstract class UnverifiedJsonWebToken {
 
-    private static final ObjectMapper MAPPER = new ObjectMapper()
+    private static final ObjectReader READER = new ObjectMapper()
             .registerModule(new Jdk8Module())
-            .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+            .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
```

Comment:
why do we ignore unknown properties?

Reply:
Allows us to add additional fields to the jwt payload in the future without breaking existing consumers.